src/components/profile/ViewAvatar.js

```diff
@@ -26,12 +26,17 @@ const ViewAvatar = props => {
     props.screenProps.push('EditAvatar')
   }
 
-  const handleClosePress = event => {
+  const handleClosePress = async event => {
```

Comment:
1. use async/await
2. the async/await here is redundant since nothing happens after the await.
3. you can parallelize the setting for the two fields to null

Reply:
The same

Reply:
use async await

Reply:
the await is still redundant

Reply:
Done. see new commits